### ISSUE TITLE
owm errors

### ISSUE THREAD

**Issue by JarbasAl** [OpenVoiceOS/ovos_api_service#2]
only one call is working, additionally lat/lon should also be supported, see [local-backend](https://github.com/OpenVoiceOS/OVOS-local-backend/blob/dev/ovos_local_backend/backend/external_apis.py) for a simple implementation, the pywon dependency can also be dropped if you take code from there

```python
import requests
from ovos_utils import ovos_service_api


class OvosWeather:
    def __init__(self):
        self.api = ovos_service_api.OVOSApiService()
        self.api.register_device()

    @property
    def uuid(self):
        return self.api.get_uuid()

    @property
    def headers(self):
        self.api.get_session_challenge()
        return {'session_challenge': self.api.get_session_token(),  'backend': 'OWM'}

    def get_weather_onecall(self, query):
        reqdata = {"lat": query.get("lat"),
                   "lon": query.get("lon"),
                   "units": query.get("units"),
                   "lang": query.get("lang")}
        url = f'https://api.openvoiceos.com/weather/onecall_weather_report/{self.uuid}'
        r = requests.post(url, data=reqdata, headers=self.headers)
        return r.json()

    def get_current(self, query):
        url = f"https://api.openvoiceos.com/weather/generate_current_weather_report/{self.uuid}/{query}"
        r = requests.get(url, headers=self.headers)
        return r.json()

    def get_hourly(self, query):
        url = f"https://api.openvoiceos.com/weather/generate_hourly_weather_report/{self.uuid}/{query}"
        r = requests.get(url, headers=self.headers)
        return r.json()

    def get_forecast(self, query):
        url = f"https://api.openvoiceos.com/weather/generate_forecast_weather_report/{self.uuid}/{query}"
        r = requests.get(url, headers=self.headers)
        return r.json()

if __name__ == "__main__":
    print(OvosWeather().get_forecast("London, GB"))
    # {'error': "forecast_at_place() missing 1 required positional argument: 'interval'"}
    print(OvosWeather().get_hourly("London, GB"))
    # {'error': "'WeatherManager' object has no attribute 'hourly_forecast'"}
    print(OvosWeather().get_current("London, GB"))
    # {'error': 'An asyncio.Future, a coroutine or an awaitable is required'}
```

**Comment by JarbasAl** [comment]
helper class in example above should probably be ported to ovos_utils directly

**Comment by JarbasAl** [comment]
fixed by #4
